cmd/tetra/metrics/print.go

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package metrics
+
+import (
+	"log/slog"
+
+	"github.com/isovalent/metricstool/pkg/metricsmd"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/spf13/cobra"
+
+	"github.com/cilium/tetragon/pkg/metrics/metricsconfig"
+)
+
+func New() *cobra.Command {
+	targets := map[string]string{
+		"health":    "Tetragon Health",
+		"resources": "Tetragon Resources",
+		"events":    "Tetragon Events",
+	}
+
+	overrides := []metricsmd.LabelOverrides{
```

Comment:
do you need to provide "false" values here or can you just pass an empty object? Can be a bit misleading

Reply:
I could pass empty strings as label values, then the docs would still list the label names but not values.

I'm thinking having example values is helpful (users can see the format), but currently the reference doesn't distinguish between example and real values, so I see how it can be misleading. Maybe the next iteration of metricstool could support rendering example and real label values differently or something.

Reply:
For the build info labels specifically, we can make them less confusing by updating them on a release too.